MarriagePaper/src/main/java/com/beauver/minecraft/plugins/marriagePaper/Commands/MarryCommand.java

```diff
@@ -592,5 +594,51 @@ public void marryList(CommandSender sender){
         }
         sender.sendMessage(marryList);
     }
+    @Subcommand("tp")
+    public void marryTp(CommandSender sender, String[] args) {
+        if(!(sender instanceof Player player)) {
+            sender.sendMessage(Component.text("Only a player can run this command.").color(TextColor.fromHexString("#FF5555")));
+            return;
+        }
+
+        Couple couple = null;
+        boolean isFirst = true;
+        for(Couple c : MarriageHandler.getMarriages()){
+            if(c.getPartner1().equals(player.getUniqueId())){
+                couple = c;
+                isFirst = false;
+                break;
+            }else if(c.getPartner2().equals(player.getUniqueId())){
+                couple = c;
+                break;
+            }
+        }
+        if(couple == null){
+            player.sendMessage(Component.text("You can not tp to the air.").color(TextColor.fromHexString("#FF5555")));
+            return;
+        }
+
+        Player target;
+        if(isFirst){
+            target = Bukkit.getPlayer(couple.getPartner1());
+        }else{
+            target = Bukkit.getPlayer(couple.getPartner2());
+        }
+        if(target == null){
+            player.sendMessage(Component.text("Your partner is not online. I'm afraid you can't tp to an unknown location.").color(TextColor.fromHexString("#FF5555")));
+            return;
+        }
+
+        int delayInSeconds = 2; // maybe add config?
+        long delayInTicks = delayInSeconds * 20L; // Convert seconds to ticks
+
+        target.sendActionBar(Component.text("Your partner will tp to you in '" + delayInSeconds +"' seconds...").color(TextColor.fromHexString("#FFAA00")));
+        player.sendActionBar(Component.text("You will be tped in  '" + delayInSeconds +"' seconds...").color(TextColor.fromHexString("#FFAA00")));
+        Bukkit.getScheduler().runTaskLater(plugin, () -> {
+            player.teleport(target.getLocation());
+            player.sendActionBar(Component.text("You have been tped to ur partner!").color(TextColor.fromHexString("#FFAA00")));
+            target.sendActionBar(Component.text("Your partner has tped to you!").color(TextColor.fromHexString("#FFAA00")));
+        }, delayInTicks);
+    }
```

Comment:
There's a static reference to the plugin in ``MarriagePaper#plugin``
variable ``delayInTicks`` is redundant and can be moved to a operation instead. As it's always a constant operation

```suggestion
        int delayInSeconds = 2; // maybe add config? -> Probs later

        target.sendActionBar(Component.text("Your partner will tp to you in '" + delayInSeconds +"' seconds...").color(TextColor.fromHexString("#FFAA00")));
        player.sendActionBar(Component.text("You will be tped in  '" + delayInSeconds +"' seconds...").color(TextColor.fromHexString("#FFAA00")));
        Bukkit.getScheduler().runTaskLater(MarriagePaper.plugin, () -> {
            player.teleport(target.getLocation());
            player.sendActionBar(Component.text("You have been tped to ur partner!").color(TextColor.fromHexString("#FFAA00")));
            target.sendActionBar(Component.text("Your partner has tped to you!").color(TextColor.fromHexString("#FFAA00")));
        }, delayInSeconds * 20L);
    }
```